typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -77,19 +81,23 @@ export const hyperlaneContextAgentChainConfig: AgentChainConfig = {
     bsc: true,
     celo: true,
     ethereum: true,
+    fraxtal: true,
     gnosis: true,
     injective: true,
     inevm: true,
+    linea: true,
     mantapacific: true,
     mode: true,
     moonbeam: true,
     // At the moment, we only relay between Neutron and Manta Pacific on the neutron context.
     neutron: false,
     optimism: true,
+    osmosis: false,
```

Comment:
left it off because we were running into issues on the validator (see dan's recent fixes), we should be good but would rather do that with a new build after we've merged this + dan's fixes